understanding/info-and-relationships.html

```diff
@@ -108,7 +108,7 @@ <h2>事例</h2>
                <li><p><strong>テキスト文書</strong></p>
                   
                   
-                  <p>構造をプログラムによる解釈が可能であるように、シンプルなテキスト文書は、タイトルの前に 2 行の空行があり、アスタリスクを使ってリスト項目を示し、その他の標準的な書式の決まりに従ってフォーマットされている。</p>
+                  <p>シンプルなテキスト文書は、タイトルの前に 2 行の空白行、リスト項目を示すアスタリスク、及びその他の標準的な書式の慣習で体裁が整えられるため、その構造はプログラムによる解釈が可能となる。</p>
```

Comment:
https://github.com/waic/wcag21/pull/1914#issuecomment-2437257779 に書きましたが事例なので「ある」を入れたいです。 
「整えられる」は「整えることが可能」という意味にも読め、ここでは例としてあげた文書が「整えられている」という話だと思いますので、「整えられている」としたいです。

```suggestion
                  <p>あるシンプルなテキスト文書は、タイトルの前に 2 行の空白行、リスト項目を示すアスタリスク、及びその他の標準的な書式の慣習で体裁が整えられているため、その構造はプログラムによる解釈が可能となる。</p>
```